### ISSUE TITLE
Code coverage result with multiple test project

### ISSUE THREAD

**Issue by jagadeesan123** [cake-build/cake#1629]
I have the one source project and this source project contains the 2 test project. In both test projects contains some list of the test cases. Please find the structure of my projects below,

Core
--------Src
--------Test
------------Test1
------------Test2

In Test1 and Test2 are the test projects which contains the Nunit test cases for the Src project. Now how I can get the code coverage result based on the above two test projects. For example, if the code coverage for Test1 is 30 and Test is 40 percent, when independently run the above cases. If i added the above two values it gives 70 percent of code coverage. But this is wrong, because for some API's both two test cases may contains the NUnit test cases.

So I need to calculate the code coverage percentage based on the two test projects. So can you please suggest how can I calculate the code coverage percentage of the source project based on the above two test projects?

**Comment by mholo65** [comment]
No, you don't need any custom calculations for multiple test projects. `NUnit` and `DotCover` should support this out of the box.

Will post simple example later when I'm near a computer.

**Comment by mholo65** [comment]
@jagadeesan123 something like this should work:

```csharp
DotCoverAnalyse((ctx) => {
        ctx.NUnit3("./test/**/bin/**/test*.dll");
    },
    "coverage.html",
    new DotCoverAnalyseSettings {
        ReportType = DotCoverReportType.HTML
    }
    .WithFilter("-:test1")
    .WithFilter("-:test2"));
```
